cmd/timestamp-server/app/serve.go

```diff
@@ -63,7 +65,7 @@ var serveCmd = &cobra.Command{
 
 		server.Host = viper.GetString("address")
 		server.Port = int(viper.GetUint("port"))
-		server.EnabledListeners = []string{"http"}
+		server.EnabledListeners = viper.GetStringSlice("scheme")
```

Comment:
FWIW, this was copied from Rekor, which only supports http.

I believe we'd need additional configuration to support HTTPS. I'd probably say to just stick to supporting HTTP and terminating the TLS connection at ingress.

Reply:
The code generator provides all the code to support https, I suggest you run `timestamp-server serve -h`. Otherwise we should also exclude those available but unsupported flags.

Yes, I saw it in Rekor too. I had the feeling https makes more sense for the TSA.

Reply:
typically you'd run with a separate ingress or proxy that terminates SSL, so thats why the listener was originally pinned to http only.

Reply:
Agreed, I think it's reasonable to support it optionally though. `scheme` defaults to `http` so HTTPS would only be with explicit configuration

Reply:
@bobcallaway @haydentherapper Yes, that setup might be common under most of organizations. It was more of a question about wether we allow them to use these flags or delete them from the available ones.